Dockerfile

```diff
@@ -5,13 +5,12 @@
 # docker run -d -p 3000:3000 flowise
 
 FROM node:18-alpine
-RUN apk add --update libc6-compat
+RUN apk add --update libc6-compat python3 make g++
```

Comment:
Are these three runtime dependencies? It can be compiled successfully even without adding.

Reply:
here - https://github.com/FlowiseAI/Flowise/issues/241